crates/biome_js_analyze/tests/specs/nursery/useConsistentArrowReturn/requireReturn.invalid.js.snap

```diff
@@ -0,0 +1,96 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+expression: requireReturn.invalid.js
+---
+# Input
+```js
+
+
+const returnsObjectArrow = () => ( { a: 1, b: 2 });
+
+
+const returnsSequenceArrow = () => {
+    return (a, b);
+}
+
+
+const returnsAwaitArrow = async () => {
+    return await fetchData();
+}
+
+
+```
+
+# Diagnostics
+```
+requireReturn.invalid.js:3:28 lint/nursery/useConsistentArrowReturn  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━
+
+  ! This arrow function body should be a block statement.
+  
+  > 3 │ const returnsObjectArrow = () => ( { a: 1, b: 2 });
+      │                            ^^^^^^^^^^^^^^^^^^^^^^^
+    4 │ 
+  
+  i Safe fix: Add braces to the arrow function body.
+  
+     1  1 │   
+     2  2 │   
+     3    │ - const·returnsObjectArrow·=·()·=>·(·{·a:·1,·b:·2·});
+        3 │ + const·returnsObjectArrow·=·()·=>·{return·{·a:·1,·b:·2·};};
```

Comment:
I feel in a use case like this we should at least make an effort to put the `return` statement on its own line.